confgenerator/logging_processors.go

```diff
@@ -275,6 +275,55 @@ func (p LoggingProcessorNestWildcard) Components(ctx context.Context, tag, uid s
 	}
 }
 
+type LoggingProcessorLift struct {
+	NestedUnder string
+	AddPrefix   string
+}
+
+func (p LoggingProcessorLift) Components(ctx context.Context, tag, uid string) []fluentbit.Component {
+	filter := fluentbit.Component{
+		Kind: "FILTER",
+		Config: map[string]string{
+			"Name":          "nest",
+			"Match":         tag,
+			"Operation":     "lift",
+			"Nested_under":  p.NestedUnder,
+			"Add_prefix":    p.AddPrefix,
+		},
+	}
+
+	return []fluentbit.Component{
+		filter,
+	}
+}
+
+type LoggingProcessorGrep struct {
+	Field   string `validate:"required"`
```

Comment:
These validate tags will never be used, so it's probably more confusing than helpful to have them here.

Reply:
I suppose you're right — I was using them as pseudo-documentation to indicate that Regex and Exclude are mutually-exclusive, but I can just say that in a comment…